cmd/loadtest/loadtest.go

```diff
@@ -667,8 +679,8 @@ func mainLoop(ctx context.Context, c *ethclient.Client, rpc *ethrpc.Client) erro
 		}
 
 		tops.Nonce = new(big.Int).SetUint64(currentNonce)
-		tops = configureTransactOpts(tops)
 		tops.GasLimit = 10000000
+		tops = configureTransactOpts(tops, legacyTransactionMode)
```

Comment:
Maybe I'm wrong but why do we need to call this function again if we already called it on line `614`?

Reply:
yeah i also think the second and third `configureTransactOpts ` calls are redundant in this case. i think it was just a safeguard of making sure we are enforcing some options.